sql_generators/glean_usage/__init__.py

```diff
@@ -99,6 +99,7 @@ def generate(
     elif exclude:
         table_filter = partial(table_matches_patterns, exclude, True)
 
+    @cache
```

Comment:
Could it be more beneficial to cache `sql_generators.glean_usage.common.list_tables` instead? This fn mostly delegates there then does a filter (unless the filter is what takes a lot of time here)

Reply:
It wouldn't make much of a difference. The only call to that `list_tables` is this one here in `get_tables`.